SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -0,0 +1,234 @@
+%define share_kata %{coco_path}/share/kata-containers
+
+# Symlinks for cc binaries
+mkdir -p %{buildroot}/%{coco_bin}
+mkdir -p %{buildroot}/%{coco_path}/libexec
+mkdir -p %{buildroot}/etc/systemd/system/containerd.service.d/
+ln -s /usr/bin/cloud-hypervisor %{buildroot}/%{coco_bin}/cloud-hypervisor
+ln -s /usr/bin/containerd %{buildroot}/%{coco_bin}/containerd
+ln -sf /usr/libexec/virtiofsd %{buildroot}/%{coco_path}/libexec/virtiofsd
+install -D -m 0644 %{SOURCE4} %{buildroot}/etc/systemd/system/containerd.service.d/containerd-for-cc-override.conf
+
+find %{buildroot}/etc
+
+# Runtime
+pushd %{_builddir}/%{name}-%{version}/src/runtime
+install -D -m 0755 containerd-shim-kata-v2 %{buildroot}/usr/local/bin/containerd-shim-kata-cc-v2
+install -D -m 0755 kata-monitor %{buildroot}/%{coco_bin}/kata-monitor
+install -D -m 0755 kata-runtime %{buildroot}/%{coco_bin}/kata-runtime
+install -D -m 0755 data/kata-collect-data.sh %{buildroot}/%{coco_bin}/kata-collect-data.sh
+
+install -D -m 0644 config/configuration-clh.toml %{buildroot}/%{defaults_kata}/configuration-clh.toml
+ln -s configuration-clh.toml %{buildroot}/%{defaults_kata}/configuration.toml
+sed -i 's|/usr|/opt/confidential-containers|g' %{buildroot}/%{defaults_kata}/configuration-clh.toml
+popd
+
+# Tardev-snapshotter
+pushd %{_builddir}/%{name}-%{version}/src/tardev-snapshotter/
+sed -i -e 's/containerd.service/kubelet.service/g' tardev-snapshotter.service
+install -m 0644 -D -t %{buildroot}%{_unitdir} tardev-snapshotter.service
+install -D -m 0755 target/release/tardev-snapshotter  %{buildroot}/usr/bin/tardev-snapshotter
+
+popd
+
+# The UVM rootfs will be used to build an img file at the first host boot.
+install -D -m 0755 %{_builddir}/%{name}-%{version}/tools/osbuilder/scripts/lib.sh                   %{buildroot}/%{share_kata}/scripts/lib.sh
+install -D -m 0755 %{_builddir}/%{name}-%{version}/tools/osbuilder/image-builder/image_builder.sh   %{buildroot}/%{share_kata}/scripts/image_builder.sh
+install -D -m 0755 %{_builddir}/%{name}-%{version}/tools/osbuilder/image-builder/nsdax              %{buildroot}/%{share_kata}/scripts/nsdax
+install -D -m 0644 %{_builddir}/%{name}-%{version}/tools/osbuilder/mariner-uvm-rootfs.tar.gz        %{buildroot}/opt/mariner/share/uvm/mariner-uvm-rootfs.tar.gz
+
+install -D -m 0755 %{SOURCE2} %{buildroot}/opt/mariner/share/uvm/mariner-coco-build-uvm-image.sh
+install -D -m 0755 %{SOURCE6} %{buildroot}/opt/mariner/share/uvm/pause-image.sh
+install -D -m 0644 %{SOURCE5} %{buildroot}/runtime.yaml
+
+install -m 0644 -D -t %{buildroot}%{_unitdir} %{SOURCE3}
+install -m 0644 -D -t %{buildroot}%{_unitdir} %{SOURCE7}
+
+%preun
+%systemd_preun tardev-snapshotter.service
+%systemd_preun mariner-coco-build-uvm-image.service
+
+%postun
+%systemd_postun tardev-snapshotter.service
+%systemd_postun mariner-coco-build-uvm-image.service
+
+%post
+%systemd_post tardev-snapshotter.service
+%systemd_post mariner-coco-build-uvm-image.service
+
+%files
+/opt/*
```

Comment:
Consider explicitly declaring the exact files you expect to package here rather than wildcard so it is clear to the reader what is and is not being packaged.

It also allows the rpmbuild tooling to help identify when unpackaged new bits make their way into the installdir on future updates

Reply:
Explicit files added under %install section